2-mandatory/1-weather-report.js

```diff
@@ -11,7 +11,18 @@
         - Hint: you can call the temperatureService function from your function
 */
 
-function getTemperatureReport(cities) {
+    function getTemperatureReport(cities) {
```

Comment:
This looks very good 👍 
Try to keep an eye on indentation and spacing. It sounds like a minor point - can make it much easier for other developers to read your code 😄